src/DbSql/TaoRdf/UnionQuerySerialyser.php

```diff
@@ -285,9 +285,8 @@ protected function orderByPart(array $aliases) {
         $sortFields = [];
 
         foreach ($aliases as $alias) {
-            $sortFields[] = $this->getDriverEscaper()->reserved($alias['name']) . '.' .
-                    $this->getDriverEscaper()->reserved('object') . ' ' .
```

Comment:
i as understand it : this will produce the following 
`ORDER BY orderq1.object ASC`
and you did the followin 
`orderq1. asc` because you have removed object 
so your removing the column that you need to order by 
instead or removing the prefix

Reply:
this will fail ,  did you test?

Reply:
Yes. It's working like that.

Reply:
I removed only `.object` from query

Reply:
I checked this on both `pgsql` and `Spanner`